### ISSUE TITLE
AWS report is incorrect

### ISSUE THREAD

**Issue by jasonkarns** [b3nj5m1n/xdg-ninja#408]
# tldr

xdg-ninja incorrectly reports the existence of `.aws/cli` and `.aws/sso` as fixable. I would hope that the aws check only fails on existence of .aws/config and .aws/credentials which _are_ fixable with env vars presently.

# details

The check for aws files seems to only be checking existence of ~/.aws directory, and reports "fixable":

```
  Export the following environment variables:                                                                         
                                                                                                                    
    export AWS_SHARED_CREDENTIALS_FILE="$XDG_CONFIG_HOME"/aws/credentials                                             
    export AWS_CONFIG_FILE="$XDG_CONFIG_HOME"/aws/config                                                              
```

However, I have those set:

```
$ echo $AWS_CONFIG_FILE 
/Users/jason.karns/.config/aws/config.chime
$ echo $AWS_CREDENTIALS_FILE 
/Users/jason.karns/.local/share/aws/credentials
$ echo $AWS_SHARED_CREDENTIALS_FILE 
/Users/jason.karns/.local/share/aws/shared-credentials
```

And none of those files exist in my homedir. I _do_ have `cli` and `sso` subdirs:
```
$ ls .aws/
cli/ sso/
```

But those are not currently fixable with env vars. 
- https://github.com/aws/aws-cli/issues/7513
- https://github.com/aws/aws-cli/issues/5802
- https://github.com/aws/aws-sdk/issues/30
